Application/registerationForm.js

```diff
@@ -0,0 +1,49 @@
+document.getElementById('registerationForm').addEventListener('submit', function (event) {
+  event.preventDefault();
+
+  if (emailCheck() && contactCheck()) {
+    alert("Registeration done sucessfull");
+    location.replace("home.html");
+  } else {
+    alert("Registeration not sucessfull");
+
+  }
+});
+
+function passwordShow() {
```

Comment:
This function name should be `showPassword`.The showPassword is more readable and meaningful. It directly conveys that the function is responsible for showing the password, making it easier for other developers (or your future self) to understand the code.